openpype/modules/clockify/clockify_module.py

```diff
@@ -129,38 +112,33 @@ def check_running(self):
                 if self.bool_timer_run is True:
                     self.clockify_timer_stopped()
                 elif self.bool_timer_run is False:
-                    actual_timer = self.clockapi.get_in_progress()
-                    if not actual_timer:
+                    current_timer = self.clockapi.get_in_progress()
+                    if current_timer is None:
                         continue
-
-                    actual_proj_id = actual_timer["projectId"]
-                    if not actual_proj_id:
+                    current_proj_id = current_timer.get("projectId")
+                    if not current_proj_id:
                         continue
 
-                    project = self.clockapi.get_project_by_id(actual_proj_id)
+                    project = self.clockapi.get_project_by_id(current_proj_id)
                     if project and project.get("code") == 501:
                         continue
 
-                    project_name = project["name"]
+                    project_name = project.get("name")
 
-                    actual_timer_hierarchy = actual_timer["description"]
-                    hierarchy_items = actual_timer_hierarchy.split("/")
+                    current_timer_hierarchy = current_timer.get("description")
```

Comment:
If there can be missing description and it was changed to save access of `"description"` then the rest of logic should be skipped.
```suggestion
                    current_timer_hierarchy = current_timer.get("description")
                    if not current_timer_hierarchy:
                        continue
```